README.md

```diff
@@ -75,7 +78,7 @@ $ python3 pacman_html_generator.py --h
 * Automate tournament using a `driver.py` script and `cron`.
     
```

Comment:
We can add "* Handling of special staff teams, including ability to run contest only against those teams"